### DISCUSSION TITLE
is it possible to support the table name and column name the same

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [diesel-rs/diesel#3086] [Q&A]
this is the problem I am facing right now:

https://stackoverflow.com/questions/70438471/error-column-article-content-cannot-be-named-the-same-as-its-table

I was wonder is it possible to support this, my table name and column name could not change.

**Comment by weiznich** [comment]
Please always provide the following information if you ask questions:

* A reproducible example of the code causing your problems
* A description of what you want to do
* A description of how that fails given the reproducible example
* The error message emitted by rustc, if the compilation fails
* General information about your setup:
    * Rustc version
    * Diesel version + feature flags
    * Operating system
    * Database system

And no, just linking to some other question and stating I have the same problem does not provide these information. If you expect that other people spend time on answering your question, you need to invest time in your questions.
